.gitattributes

```diff
@@ -0,0 +1,304 @@
+# Standard gitattributes config
+
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text eol=lf
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+
+*.txt       text
+
+# Source code
+*.bash      text eol=lf
+*.c         text
+*.cpp       text
+*.csh       text eol=lf
+*.fish      text eol=lf
+*.inc       text
+*.ipynb     text
+*.h         text
+*.ksh       text eol=lf
+*.ps1       text
+*.pxd       text diff=python
+*.py        text diff=python
+*.py3       text diff=python
+*.pyi       text diff=python
+*.pyw       text diff=python
+*.pyx       text diff=python
+*.qss       text
+*.r         text
+*.R         text
+*.rb        text
+*.rmd       text
+*.Rmd       text
+*.rnw       text
+*.Rnw       text
+*.sh        text
+*.zsh       text eol=lf
+
+# Documentation
+*.adoc      text
+*.latex     text
+*.LaTeX     text
+*.markdown  text
+*.md        text
+*.po        text
+*.pot       text
+*.rd        text
+*.Rd        text
+*.rst       text
+*.tex       text
+*.TeX       text
+*.tmpl      text
+*.tpl       text
+
+# Web
+*.atom      text
+*.css       text
+*.htm       text
+*.html      text
+*.js        text
+*.jsx       text
+*.json      text
+*.php       text
+*.pl        text
+*.rss       text
+*.sass      text
+*.scss      text
+*.xht       text
+*.xhtml     text
+
+# Configuration
+*.cfg       text
+*.cnf       text
+*.conf      text
+*.config    text
+*.desktop   text
+*.inf       text
+*.ini       text
+*.plist     text
+*.toml      text
+*.xml       text
+*.yml       text
+*.yaml      text
+
+# Plain text data
+*.cdl       text
+*.csv       text
+*.dif       text
+*.geojson   text
+*.gml       text
+*.kml       text
+*.sql       text
+*.tab       text
+*.tsv       text
+*.wkt       text
```

Comment:
Is this `Plain text data` section necessary here?

Reply:
See [my comment](https://github.com/spyder-ide/qtpy/pull/278#issuecomment-963716528)